server/command.go

```diff
@@ -193,23 +193,26 @@ func (p *Plugin) runListCommand(args []string, extra *model.CommandArgs) (*model
 }
 
 func (p *Plugin) runPopCommand(args []string, extra *model.CommandArgs) (*model.CommandResponse, bool, error) {
-	todoMessage, sender, err := p.listManager.PopIssue(extra.UserId)
+	issue, err := p.listManager.PopIssue(extra.UserId)
 	if err != nil {
 		return nil, false, err
 	}
 
-	if sender != "" {
-		userName := p.listManager.GetUserName(extra.UserId)
+	userName := p.listManager.GetUserName(extra.UserId)
 
-		message := fmt.Sprintf("@%s popped a Todo you sent: %s", userName, todoMessage)
-		p.sendRefreshEvent(sender)
-		p.PostBotDM(sender, message)
+	if issue.ForeignUser != "" {
+		message := fmt.Sprintf("@%s popped a Todo you sent: %s", userName, issue.Message)
+		p.sendRefreshEvent(issue.ForeignUser)
+		p.PostBotDM(issue.ForeignUser, message)
 	}
 
 	p.sendRefreshEvent(extra.UserId)
 
 	responseMessage := "Removed top to do."
 
+	replyMessage := fmt.Sprintf("@%s popped a todo attached to this thread", userName)
```

Comment:
nit: What's the right format, `todo` or `Todo` (or `TODO`?)

Reply:
The other comments are addressed. I am asking on the todo plugin channel which option looks better, and I will substitute every mention with the final decision.